docker/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM docker.io/library/ubuntu:focal
+FROM docker.io/library/ubuntu:jammy
 
 ARG DEBIAN_FRONTEND=noninteractive
 ENV TZ=Etc/UTC
-RUN apt-get update && \
+RUN apt-get update &&
     apt-get install -y \
         build-essential \
-        g++ clang-10 libc++-10-dev libc++abi-10-dev cmake ninja-build libx11-dev libxcursor-dev libxi-dev libgl1-mesa-dev libfontconfig1-dev \
+        g++ clang cmake ninja-build libx11-dev libxcursor-dev libxi-dev libgl1-mesa-dev libfontconfig1-dev \
```

Comment:
Nice! I previously needed to manually install `clang-10` package on `focal` just because the version of the "normal" `clang` package was too old. Since there is a new `jammy` LTS now with a recent enough `clang` - this makes makes more sense.